src/main/java/elysia/ui/Ui.java

```diff
@@ -0,0 +1,85 @@
+package elysia.ui;
+
+import elysia.command.Task;
+
+import java.util.ArrayList;
+
+/**
+ * Text UI of the application.
+ */
+public class Ui {
+    private static String line = "____________________________________________________________";
+    private static String welcomeMessage = "Hi there! Did you miss me?\n" +
+            "Wherever you are and whenever you need,\n" +
+            "Elysia will always meet your expectations.";
+    private static String exitMessage = "Alright, this time we really have to say goodbye.\n" +
+            "Goodbye, Mei!";
```

Comment:
Following the coding standard for this class, the break should be before `+`
ie. 
```java
private static String welcomeMessage = "Hi there! Did you miss me?\n"
            + "Wherever you are and whenever you need,\n"
            + "Elysia will always meet your expectations.";
```
other than that, good break point! Easy to visualise the output of the program. 

Side note: personally, I do prefer what you have done (break after `+`) as it tells me I have to continue reading next line like a "go-to-next-page" but oh well, just gotta follow the rules like prof said :)